### ISSUE TITLE
Don't work with domain config

### ISSUE THREAD

**Issue by lamvnd** [laravel/telescope#590]
I want to config telescope on subdomain. Like this: monitor.example.local

When i enter 'domain' config item on config file. like this: 'domain' => 'monitor.example.local',

telescope don't work like well.

**Comment by themsaid** [comment]
You mean you receive 404 or what?

**Comment by lautaroml** [comment]
Don´t work under sub domain.

I have a subdomain: dashboard.mysite.com

When I enter in dashboard.mysite.com/telescope the telescope page loads but telescope watchers seems to not register anything....

I clicked on every section (watcher) of the telescope page but they all through "We didn't find anything - just empty space."

Telescope config:
'domain' => env('TELESCOPE_DOMAIN', 'dashboard.mysite.com')
'path' => env('TELESCOPE_PATH', 'telescope')

**Comment by base64bits** [comment]
This seems to be the same issue I have faced after moving to production, here are my details:

When in local environment Telescope works as expected, when I moved to production telescope saves the entries in the database but in the browser it keeps scanning and entries never load.

I have Laravel installed in a subdomain with SSL working.

In Safari when I look at the network tab and reload the page I get an error for loading the entries:

xhr failed to load resource: The network connection was lost

If I open the page in FireFox the entries load fine but they dont auto load with the auto load entries button clicked, I have to manually refresh the page to see new entries.

I have uninstalled Telescope, installed it again, clear the DB tables of all the entries and nothing seems to work.

This is under Laravel 7.1.3 and Telescope 3.1
